lib/features/drilldown/DrilldownOverlayBehavior.js

```diff
@@ -169,15 +172,19 @@ DrilldownOverlayBehavior.prototype._updateOverlayVisibility = function(element)
  */
 DrilldownOverlayBehavior.prototype._addOverlay = function(element) {
   var canvas = this._canvas,
-      overlays = this._overlays;
+      overlays = this._overlays,
+      bo = getBusinessObject(element);
 
   var existingOverlays = overlays.get({ element: element, type: 'drilldown' });
 
   if (existingOverlays.length) {
     this._removeOverlay(element);
   }
 
-  var button = domify('<button class="bjs-drilldown">' + ARROW_DOWN_SVG + '</button>');
+  var button = domify('<button type="button" class="bjs-drilldown">' + ARROW_DOWN_SVG + '</button>'),
+      elementName = bo.get('name') || bo.get('id'),
+      title = this._translate('Drill down {element}', { element: elementName });
```

Comment:
Shall this be `Drill down _into_ {element}`. I'd not drill down `ServiceTask`.

Reply:
I see. What if we called it "Open Service Task"?

Reply:
I think drill [down is the word](https://www.merriam-webster.com/dictionary/drill%20down). What would be the benefit of open?

Reply:
I think "open" is much more common and therefore easier to understand. "Drill down" has always sounded unfamiliar to me.
@lmbateman Could you share your view on this?

Reply:
A couple of things to consider:

- "Drill down" may imply a deeper level of analysis than "open":
- - the [Merriam-Webster dictionary](https://www.merriam-webster.com/dictionary/drill%20down) defines it as "to thoroughly consider or analyze : [EXAMINE](https://www.merriam-webster.com/dictionary/examine)". 
- - The [Oxford Learner's dictionary](https://www.oxfordlearnersdictionaries.com/us/definition/english/drill-down) defines it as, "(computing or business) to go to deeper levels of an organized set of data in order to find more detail, especially on a computer or a website".
- Typically we open links and documents, but those are usually considered peers. "Drill down" to me definitely implies a child, whereas I think "open" could refer to either a peer or child.
- My sense is that "open" might be easier for non-native speakers, and might work better for internationalization.
- I feel like I would know what "open" does, but might have a slight moment of hesitation about what "drill down" does.

Overall, I have a preference for "open" over "drill down", but not if we think "open" loses too much precision.

Reply:
Thanks Leah for your insights. I will use "open" then, but we can always change it to "drill down" if there's strong need.

Reply:
BTW we already use "Open" in dmn-js: https://github.com/bpmn-io/dmn-js/pull/843/files#diff-b5845c79db311a2ace41bb7197593074737bc342d8974b6d4069315ae7ced9ae

Reply:
Ok, so we go for "open XYZ" consistently across our tools now. I'm OK with this though drilldown in the computing or business sense makes 100% sense.

> The [Oxford Learner's dictionary](https://www.oxfordlearnersdictionaries.com/us/definition/english/drill-down) defines it as, "(computing or business) to go to deeper levels of an organized set of data in order to find more detail, especially on a computer or a website".